### PR TITLE
Add balances documentation

### PR DESCRIPTION
**Pull request by mischasteiner73**
This PR adds the documentation about the balances api endpoint. Related: https://github.com/pia-planer/pia-app/pull/271

### CODE REVIEW

File: positions/balances.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+curl GET https://app.pia-planer.ch/api/v1/positions/:position_id/balances?date=2021-05-31 \
+   -H 'Authorization: Business YOUR_API_KEY'
+```
+
+### Response
+
+Response is an `Object`.
+
+```json
+[
```
Comment (mischasteiner73) [inline positions/balances.md:43]: Array weg

File: positions/balances.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  {
+    "effective_target_hours": "560.25",
```
Comment (mischasteiner73) [inline positions/balances.md:45]: " weg

**Comment by mischasteiner73** [comment]
@lxxxvi 

This should be now ready...

File: positions/balances.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+#### Item attributes
+
+| Key                                | Type     | Can be null? | Description                                                    | Example values |
+| ---------------------------------- | -------- | ------------ | -------------------------------------------------------------- | -------------- |
+| `target_working_hours`             | `number` | no           | Target working hours as decimal number                         | `560.25`       |
+| `actual_working_hours`             | `number` | no           | Actual working hours as decima number                          | `555.00`       |
+| `working_hours_balance`            | `number` | no           | Difference of target and actual working hours                  | `-5.25`        |
+| `total_vacation_hours_in_year`     | `number` | no           | Total vacations in year in hours as decimal number             | `200.00`       |
```
Comment (lxxxvi) [inline positions/balances.md:64]:
```suggestion
| `total_vacation_hours_in_year`     | `number` | no           | Total vacation hours in year as decimal number             | `200.00`       |
```

File: positions/balances.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+| `planned_vacation_hours_in_year`   | `number` | no           | Planned vacations in year in hours as decimal number           | `180.5`        |
```
Comment (lxxxvi) [inline positions/balances.md:65]:
```suggestion
| `planned_vacation_hours_in_year`   | `number` | no           | Planned vacation hours in year as decimal number           | `180.5`        |
```

**Review by lxxxvi — APPROVED** [review]
@mischasteiner73 I added two small suggestions, feel free to commit them.

**Pull request merged**